### DISCUSSION TITLE
How to uninstall?

### DISCUSSION THREAD

**Discussion by cameo69** [joyfullservice/msaccess-vcs-addin#203] [General]
Hi, could somebody point me to an explanation how to deinstall the tool?
I like it and I like to keep it, but I have currently two different versions installed and do not know how to get rid of the older one.
The project Wiki https://github.com/joyfullservice/msaccess-vcs-integration/wiki/Installation does not seem to contain information about deinstallation.

**Comment by cameo69** [comment]
Oh dear, just found it here at the bottom: https://github.com/joyfullservice/msaccess-vcs-integration/wiki/Documentation
Uninstall via Options --> Settings --> Uninstall

**Reply by hecon5** [reply]
You do bring up a good point; I've edited the install page to link to the removal tool.

Out of curiosity...how did you install two versions? The tool usually tries to update, not duplicate the installation. This may be a bug.

**Reply by cameo69** [reply]
I can tell you what I did. Where it got derailed, no clue.
Before I updated, I had installed v3.1.56. Then I checked for an update on the github and downloaded v3.3.1, extracted the zip, and installed it the usual way (opening and click install).
That's how I ended up with both version installed. Later I found the button to uninstall and pressed within the settings of v3.1.56 and it removed itself.

**Reply by hecon5** [reply]
Interesting! I thought we'd worked to ensure that the old one was removed on 3.3.1. 

FWIW, I updated the wiki to add uninstall instructions on the install page; thanks for bringing it up!
Looks like going this far forward may not be captured in checking for prior versions. I'll make an issue.

**Reply by cameo69** [reply]
One more thing, the names in the menu were different in v3.1.56.
Now the menu says "VCS Export All Source", but the old one was something like Export Source, definitely without the prefix VCS. Maybe that's why it did not find the old installation.

**Reply by hecon5** [reply]
Well, the system should check for the old names and versions; it might not be checking though. I've made an issue joyfullservice#204, to address this. If you have any other info, please respond over there so we can capture that :)
